website/src/pages/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import * as icons from "../icons";
+
+import styles from "./index.module.css";
+
+type FeatureItem = {
+  title: string;
+  Icon: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+  description: string;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: "Pub/Sub logging",
+    Icon: icons.Robot,
+    description:
+      "MCAP is ideal for storing multiple channels of timestamped log data, such as pub/sub messages or multimodal sensor data.",
+  },
+
+  {
+    title: "Serialization agnostic",
+    Icon: icons.DrawerEnvelope,
+    description:
+      "MCAP is a container format, allowing you to record and replay binary messages in any format, such as Protobuf, DDS (CDR), ROS, JSON, etc.",
+  },
+  {
+    title: "High performance writing",
```

Comment:
```suggestion
    title: "High-performance writing",
```

Reply:
is that correct? why would you hyphenate it?

Reply:
https://www.grammarly.com/blog/hyphen-with-compound-modifiers/

Reply:
TIL